.ci/install_crio.sh

```diff
@@ -18,9 +18,13 @@ set -e
 
 cidir=$(dirname "$0")
 
+source "${cidir}/../test-versions.txt"
+
 echo "Get CRI-O sources"
 go get -d github.com/kubernetes-incubator/cri-o || true
 pushd $GOPATH/src/github.com/kubernetes-incubator/cri-o
+git fetch
```

Comment:
@sboeuf  @grahamwhaley  I consider the case someone run `setup` locally but cri-o was already cloned but in an old version. we need to fetch and checkout to the commit we want. In the case of semaphore all is cloned in the moment. but if this setup will run only in semaphore I can remove it.